qualtran/bloqs/data_loading/qrom.py

```diff
@@ -54,7 +52,7 @@ def _to_tuple(x: Iterable[NDArray]) -> Sequence[NDArray]:
 
 @cirq.value_equality()
 @attrs.frozen
-class QROM(UnaryIterationGate):
+class QROM(QROMABC, UnaryIterationGate):
```

Comment:
Would it make sense to ever just have a single QROM block where you'd get QROAM if block_size > 1, or would it make things more complicated.